integ-test/src/test/java/org/opensearch/sql/sql/MatchPhraseIT.java

```diff
@@ -6,9 +6,12 @@
 
 package org.opensearch.sql.sql;
 
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ACCOUNT;
```

Comment:
Are all these imports used?

Reply:
Oops. Added because I thought I was going to use them but didn't. I will remove them.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/182/commits/2376b55fad487e7014242d3758014beb9e1e2f5c